### PR TITLE
Update Governance links from HEAD to main

### PR DESCRIPTION
**Pull request by joesepi**
I am finding that when I click on a link and then want to edit that file, I am viewing it at a commit rather than on the main branch. This makes it so that the file appears uneditable. You simply need to switch to the main branch and then can edit but it may be confusing to folks.

### CODE REVIEW

**Comment by ljharb** [comment]
hm, i wonder if there's a way we could get github to make "default branch" links, so we don't have to hardcode the branch name.

File: GOVERNANCE.md [hunk 1]
```diff
@@ -231,15 +231,15 @@ The pull request may be fast-tracked if two CPC members approve the fast-trackin
 CPC members may request fast-tracking of pull requests they did not author. In that case only, the request itself is also one fast-track approval. Upvote the comment anyway to avoid any doubt.
 
 [cpc repo]: https://github.com/openjs-foundation/cross-project-council
-[cpc charter]: https://github.com/openjs-foundation/cross-project-council/blob/HEAD/CPC-CHARTER.md
-[cpc charter term]: https://github.com/openjs-foundation/cross-project-council/blob/HEAD/CPC-CHARTER.md#voting-members
-[CPC charter section 5]: https://github.com/openjs-foundation/cross-project-council/blob/HEAD/CPC-CHARTER.md#section-5-responsibilities-and-expectations-of-the-cpc
+[cpc charter]: https://github.com/openjs-foundation/cross-project-council/blob/main/CPC-CHARTER.md
+[cpc charter term]: https://github.com/openjs-foundation/cross-project-council/blob/main/CPC-CHARTER.md#voting-members
+[CPC charter section 5]: https://github.com/openjs-foundation/cross-project-council/blob/main/CPC-CHARTER.md#section-5-responsibilities-and-expectations-of-the-cpc
```
Comment (eemeli) [inline GOVERNANCE.md:236]:
```suggestion
[cpc charter]: ./CPC-CHARTER.md
[cpc charter term]: ./CPC-CHARTER.md#voting-members
[CPC charter section 5]: ./CPC-CHARTER.md#section-5-responsibilities-and-expectations-of-the-cpc
```

Why not use relative links within this repo?

**Review by mhdawson — APPROVED** [review]
LGTM

**Review by ssua91 — APPROVED** [review]

**Review by m0dul3 — APPROVED** [review]

**Review by eemeli — APPROVED** [review]

**Comment by joesepi** [comment]
This has been open for a while and while it is a change to the governance docs, it is a minor link tweak, does not change governance in any substantive way. Merging....

**Pull request merged**